js/popup.js

```diff
@@ -0,0 +1,24 @@
+$('.gameLevel').on('collision', function (player) {
```

Comment:
Function incomplete, this might be due to dependencies.

Reply:
function is ready for integration into collision detection code not yet written.